packages/components/src/components/install-guide/record-instructions/JavaScript.vue

```diff
@@ -0,0 +1,147 @@
+        <p>
+          When your application uses {{ this.webFramework.name }}, and you run your application with
+          AppMap enabled, HTTP server request recording will be enabled. To record requests, first
+          run your application by passing your launch command as the arguments to
+          <code class="inline">npx appmap-node</code>.
+        </p>
+        <p>
+          For example, if you normally start your server with
+          <code class="inline">npm run dev</code>:
+          <v-code-snippet clipboard-text="npx appmap-node npm run dev" :kind="codeSnippetType" />
+        </p>
+        <p>
+          Interact with your application, through its user interface and/or by making API requests
+          using a tool such as Postman. An AppMap will be created for each HTTP server request
+          that's served by your app.
+        </p>
+        <p>
+          For more information, visit
+          <a
+            href="https://appmap.io/docs/reference/appmap-node.html#request-recording"
+            target="_blank"
+            >AppMap docs — Node.js Requests recording</a
+          >.
+        </p>
+      </section>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><RemoteRecordingIcon /></i>Remote recording
+        </h3>
+        <p>
+          When your application uses {{ this.webFramework.name }}, and you run your application with
+          AppMap enabled, remote recording is enabled. To make a remote recording, run your
+          application using <code class="inline">npx appmap-node</code> command. Then use the
+          "Record" button in the IDE to start recording. Interact with your application, through its
+          user interface and/or by making API requests using a tool such as Postman. When you are
+          done, click the "Record" button again to save the recording and view the AppMap.
+        </p>
+        <p>
+          For more information, visit
+          <a
+            href="https://appmap.io/docs/reference/appmap-node.html#remote-recording"
+            target="_blank"
+            >AppMap docs — Node.js Remote recording</a
+          >.
+        </p>
+      </section>
+    </template>
+    <template v-else>
+      <section class="recording-method recording-method--disabled">
+        <h3>
+          <i class="header-icon header-icon--disabled"><RequestsIcon /></i>HTTP recording
+        </h3>
+        <p>
+          Did you know? When you run an express.js, nest.js or next.js app, you can make AppMaps of
+          all the HTTP requests served by your app. These weren't detected in this project, though.
+        </p>
+        <p>
+          Note most projects serving HTTP using node <i>http</i> module directly will also work,
+          even though we're not currently detecting them. For more information, see
+          <a href="https://appmap.io/docs/reference/appmap-node.html" target="_blank"
+            >AppMap for Node.js documentation</a
```

Comment:
I went to the docs but I didn't really see more information about how most projects serving HTTP will work.

Reply:
I linked instead to sections about remote and request recording.